scaletest/templates/scaletest-runner/main.tf

```diff
@@ -693,26 +855,24 @@ resource "kubernetes_pod" "main" {
         }
       }
       resources {
-        # Set requests and limits values such that we can do performant
-        # execution of `coder scaletest` commands.
         requests = {
           "cpu"    = "250m"
           "memory" = "512Mi"
         }
-        limits = {
-          "cpu"    = "${local.cpu}"
-          "memory" = "${local.memory}Gi"
-        }
       }
       volume_mount {
         mount_path = "/home/coder"
         name       = "home"
         read_only  = false
       }
-      port {
-        container_port = 21112
-        name           = "prometheus-http"
-        protocol       = "TCP"
+      dynamic "port" {
+        for_each = data.coder_parameter.load_scenario_run_concurrently.value ? jsondecode(data.coder_parameter.load_scenarios.value) : [""]
+        iterator = it
+        content {
+          container_port = 21112 + it.key
+          name           = "prom-http${it.key}"
```

Comment:
This will end up in the prom metric tags under `endpoint` and it may not be easy to correlate `endpoint:prom-http1`, `endpoint:prom-http2` etc. with the relevant scenario. Is there a different identifier we could use here?

Reply:
I'm open to suggestions, but this name has a 15 char limit so it's not easy to fit more meaning into it.